airbyte_cdk/sources/streams/concurrent/cursor.py

```diff
@@ -299,7 +301,7 @@ def ensure_at_least_one_state_emitted(self) -> None:
         """
         self._emit_state_message()
 
-    def generate_slices(self) -> Iterable[Tuple[CursorValueType, CursorValueType]]:
+    def stream_slices(self) -> Iterable[StreamSlice]:
```

Comment:
This change is going to be breaking for source-salesforce as the slicing relies on `generate_slices`

Reply:
The interfaces under the Concurrent CDK are often marked as experimental. I'll set a minor version and update source-salesforce right after